Source/Core/DolphinQt2/Settings/AudioPane.cpp

```diff
@@ -0,0 +1,236 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/AudioPane.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QFormLayout>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QRadioButton>
+#include <QSlider>
+#include <QSpacerItem>
+#include <QSpinBox>
+#include <QVBoxLayout>
+
+#include "AudioCommon/AudioCommon.h"
+#include "Core/ConfigManager.h"
+#include "DolphinQt2/Config/SettingsWindow.h"
+#include "DolphinQt2/Settings.h"
+
+AudioPane::AudioPane()
+{
+  CreateWidgets();
+  LoadSettings();
+  ConnectWidgets();
+
+  connect(&Settings::Instance(), &Settings::VolumeChanged, this, &AudioPane::OnVolumeChanged);
+}
+
+void AudioPane::CreateWidgets()
+{
+  auto* dsp_box = new QGroupBox(tr("DSP Emulator Engine"));
+  auto* dsp_layout = new QVBoxLayout;
+  auto* dsp_layout_combo = new QVBoxLayout;
```

Comment:
I agree that it looks funky, but let's push off that design change for another PR / maybe fix DolphinWX at the same time.